src/routes/api/db-migrations/db-migration-down.ts

```diff
@@ -53,7 +53,7 @@ const validateScriptAgainstDB = async (targetScript: string) => {
 	if (lastScript.length < 1) {
 		throw {
 			statusCode: 500,
-			message: `There're no scripts in db to migration down, stop proceeding. \n ${lastScript}`
+			message: `There're no scripts in db to migration down, stop proceeding. \n ${lastScript.toString()}`
```

Comment:
nit: Hmm.... if lastScript.length < 1.... does it make sense to include it into the message at all?